### PR TITLE
update readme for controllermanager chart

### PR DESCRIPTION
**Pull request by my-git9**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Update readme for controllermanager chart

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: manifests/charts/cloudcore/values.yaml [hunk 1]
```diff
@@ -126,7 +126,7 @@ controllerManager:
   resources:
     limits:
       cpu: 200m
-      memory: 50Mi
+      memory: 100Mi
     requests:
       cpu: 100m
-      memory: 25Mi
+      memory: 50Mi
```
Comment (gy95) [inline manifests/charts/cloudcore/values.yaml:132]: if modify this, better also fix values in https://github.com/kubeedge/kubeedge/blob/master/manifests/profiles/version.yaml
these two files need keeping the same with each other generally

**Comment by gy95** [comment]
LGTM

**Comment by my-git9** [comment]
PTAW @fisherxu @zc2638 @zhu733756

**Comment by my-git9** [comment]
@zhu733756 @fisherxu Can you approve it? Thanks!

**Pull request closed without merging**